lib/taskana-core/src/test/java/acceptance/task/transfer/TransferTaskAccTest.java

```diff
@@ -435,4 +435,220 @@ void should_NotSetTheTransferFlagWithinBulkTransfer_When_SetTransferFlagNotReque
+  @Test
+  void should_SetOwnerAndNotBeAuthorized_When_TransferringTask() throws Exception {
+    taskService.transferWithOwner(
+        "TKI:000000000000000000000000000000000021",
+        "WBI:100000000000000000000000000000000005",
+        "teamlead-1");
+
+    Task transferredTask = taskService.getTask("TKI:000000000000000000000000000000000021");
+    assertThat(transferredTask).isNotNull();
+    assertThat(transferredTask.isTransferred()).isTrue();
+    assertThat(transferredTask.isRead()).isFalse();
+    assertThat(transferredTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(transferredTask.getOwner()).isEqualTo("teamlead-1");
+    assertThat(transferredTask.getWorkbasketSummary().getId())
+        .isEqualTo("WBI:100000000000000000000000000000000005");
+  }
+
+  @WithAccessId(user = "teamlead-1", groups = GROUP_1_DN)
+  @Test
+  void should_BulkTransferOnlyValidTasksAndSetOwner_When_SomeTasksToTransferCauseExceptions()
+      throws Exception {
+    final Workbasket wb =
+        taskanaEngine
+            .getWorkbasketService()
+            .getWorkbasket("WBI:100000000000000000000000000000000009");
+    final Instant before = Instant.now().truncatedTo(ChronoUnit.MILLIS);
+    List<String> taskIdList =
+        Arrays.asList(
+            "TKI:000000000000000000000000000000000007", // working
+            "TKI:000000000000000000000000000000000041", // NotAuthorized READ
+            "TKI:000000000000000000000000000000000041", // NotAuthorized READ
+            "TKI:200000000000000000000000000000000008", // NotAuthorized TRANSFER
+            "", // InvalidArgument
+            null, // InvalidArgument
+            "TKI:000000000000000000000000000000000099", // not existing
+            "TKI:100000000000000000000000000000000006"); // already completed
+
+    BulkOperationResults<String, TaskanaException> results =
+        taskService.transferTasksWithOwner(
+            "WBI:100000000000000000000000000000000009", taskIdList, "teamlead-1");
+
+    // check for exceptions in bulk
+    assertThat(results.containsErrors()).isTrue();
+    assertThat(results.getErrorMap().values()).hasSize(6);
+    assertThat(results.getErrorForId("TKI:000000000000000000000000000000000041").getClass())
+        .isEqualTo(NotAuthorizedOnWorkbasketException.class);
+    assertThat(results.getErrorForId("TKI:200000000000000000000000000000000008").getClass())
+        .isEqualTo(NotAuthorizedOnWorkbasketException.class);
+    assertThat(results.getErrorForId("TKI:000000000000000000000000000000000099").getClass())
+        .isEqualTo(TaskNotFoundException.class);
+    assertThat(results.getErrorForId("TKI:100000000000000000000000000000000006").getClass())
+        .isEqualTo(InvalidTaskStateException.class);
+    assertThat(results.getErrorForId("").getClass()).isEqualTo(TaskNotFoundException.class);
+    assertThat(results.getErrorForId(null).getClass()).isEqualTo(TaskNotFoundException.class);
+
+    // verify valid requests
```

Comment:
Do we need this comment here? The test logic seems understandable without this comment

Reply:
Ok I will delete it